cli/command/service/opts_test.go

```diff
@@ -250,3 +251,47 @@ func TestHealthCheckOptionsToHealthConfigConflict(t *testing.T) {
 	_, err := opt.toHealthConfig()
 	assert.Error(t, err, "--no-healthcheck conflicts with --health-* options")
 }
+
+func TestSecretOptionsSimple(t *testing.T) {
+	var opt SecretOpt
+
+	testCase := "source=/foo,target=testing"
```

Comment:
Yeah, one thing that would help is if you could just pick a secret to use for all the examples, a real one that looks like an encrypted password or something. You could show the plain text string and how you encrypted it, and then how to use it.